### PR TITLE
<input> page title

### PR DESCRIPTION
**Pull request by estelle**
made it match other elements like select and textarea

### CODE REVIEW

File: files/en-us/web/html/element/input/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "<input>: The Input (Form Input) element"
+title: "<input>: The HTML Input element"
```
Comment (chrisdavidmills) [inline files/en-us/web/html/element/input/index.md:2]:
```suggestion
title: "<input>: The HTML Form Input element"
```

Very minor and nitpicky really; I just wondered if including "Form" might make it clearer to complete beginners who might stumble upon this page where this element is used, or what it is concerned with.

**Review by chrisdavidmills — APPROVED** [review]
@estelle minor suggestion for you; take or leave it ;-)

Approving.

File: files/en-us/web/html/element/input/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/html/element/input/index.md:2]: none of the others (fieldset, select, textarea) have it, so let's leave it off.

Comment (chrisdavidmills) [reply files/en-us/web/html/element/input/index.md:2]: OK; WFM!

**Pull request merged**